packages/react-components/src/components/VideoTile.tsx

```diff
@@ -219,24 +228,25 @@ export const VideoTile = (props: VideoTileProps): JSX.Element => {
           })}
         />
 
-        {isVideoRendered ? (
+        <Stack className={mergeStyles(videoContainerStyles, { zIndex: _VIDEO_TILE_Z_INDEX })}>
+          {onRenderPlaceholder ? (
+            onRenderPlaceholder(userId ?? '', placeholderOptions, DefaultPlaceholder)
+          ) : (
+            <DefaultPlaceholder {...placeholderOptions} />
+          )}
+        </Stack>
+
+        {isRenderElementAvailable && (
```

Comment:
A render element is a "Component with the video stream." 

I'm voicing my thoughts here, but my understanding is:
In this case, for ringing a PSTN user, no video stream would be passed but a component with some state indicating if it is ringing would be rendered. 

While for calling an ACS user, it would be ringing, then when they pick up the call with their camera on a video stream would be rendered.

If this is the case, should we update the comment on line 64 for renderElement as a video stream is not always guaranteed? And the fact that you renamed isVideoRendered to isRenderElementAvailable leads me to believe this is the case.